okonomiyaki/versions/metadata_version.py

```diff
@@ -3,7 +3,7 @@
 from ..errors import InvalidMetadataVersion
 
 
-_VERSION_R = re.compile("(?P<major>\d+)\.(?P<minor>\d+)$")
+_VERSION_R = re.compile("(?P<major>\d+)\.(?P<minor>\d+)$")  # noqa: W605
```

Comment:
w605 should not be ignored because they will become errors in future versions of python 
https://lintlyci.github.io/Flake8Rules/rules/W605.html